src/Middleware/UserPageDecorationMiddleware.php

```diff
@@ -89,7 +89,10 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
         // I don't think anyone uses this. It's not really supported
         $tpl_data['css'] = $this->Config->getSetting('css');
 
-        $tpl_data['subtest'] = $request->getAttribute("pageclass")->page;
+        // Some page might call without pageclass
+        if (!is_null($request->getAttribute("pageclass"))) {
+            $tpl_data['subtest'] = $request->getAttribute("pageclass")->page;
+        }
```

Comment:
This doesn't feel like it's solving at the right level.. why are some requests getting here without pageclass defined?

Reply:
> why are some requests getting here without pageclass defined?
Because in the `module.class` or the timepoint_list, the logic first try to load the page without being going through the middleware layers, if that logic fails to get a page, then the system tries to load the correct page. It is too strange to see that kind of logic, so every page will being rendered two times because the first won't go through. I do not understand why initially we wanted to do that, so I just let it be like what I mentioned early.